heketi-start.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# Author: Delweng Zheng <delweng@gmail.com>
+set -e
+
+usage() {
+    echo "Usage: $0 -c /path/to/config/file -b /path/to/backup/dir -d /path/to/heketi/database/dbfile"
+    exit 1
+}
+
+
+TEMP=$(getopt -o c:b:d:h --long config:,backup:,datadb:,help -- "$@")
+eval set -- "$TEMP"
+
+while true
+do
+    case $1 in
+        -c|--config)
+            config=$2; shift 2;;
+        -b|--backup)
+            backup=$2; shift 2;;
+        -d|--datadb)
+            datadb=$2; shift 2;;
+        --)
+            shift; break;;
+        -h|--help)
+            usage;;
+    esac
+done
+
+config=${config-/etc/heketi/heketi.json}
+backup=${backup-/backupdb/}
+datadb=${datadb-/var/lib/heketi/heketi.db}
+datadb_dir=$(dirname $datadb)
+
+mkdir -p $backup
+if [ ! -f $datadb ]; then
+    recent=$(ls $backup | sort -r | head -1)
+    if [ "$recent" != "" ]; then
+        tar zxf $backup/$recent -C $datadb_dir
+        ret=$?
```

Comment:
any error will cause script exit when `set -e`, for this script, `ret=$?` will never reach if `tar zxf $backup/$recent -C $datadb_dir` failed. So `set -e` may not need here.